src/adapter/graphql/src/scalars/odf_generated.rs

```diff
@@ -157,8 +157,8 @@ pub struct DataSlice {
     pub size: u64,
 }
 
-impl From<odf::DataSlice> for DataSlice {
-    fn from(v: odf::DataSlice) -> Self {
+impl From<odf::metadata::DataSlice> for DataSlice {
+    fn from(v: odf::metadata::DataSlice) -> Self {
```

Comment:
Comment for readers:
`odf::DataSlice` & `odf::metadata::DataSlice` are the same structure